ghost/stack.yml

```
@@ -18,9 +18,17 @@ services:
       url: http://localhost:8080
       # contrary to the default mentioned in the linked documentation, this image defaults to NODE_ENV=production (so development mode needs to be explicitly specified if desired)
       #NODE_ENV: development
+    volumes:
+     - some-ghost-data:/var/lib/ghost/content
 
   db:
     image: mysql:8.0
     restart: always
     environment:
       MYSQL_ROOT_PASSWORD: example
+    volumes:
+      - some-ghost-mysql:/var/lib/mysql
+
+volumes:
+  - some-ghost-data:
+  - some-ghost-mysql:
```

`docker compose` requires `volumes` to be a map, not a list of maps.
```suggestion
  some-ghost-data:
  some-ghost-mysql:
```

---

Ah good catch, I tweaked my own to become "minimal" and messed that up